nucypher/characters/control/interfaces.py

```diff
@@ -192,7 +192,8 @@ def retrieve(self,
                  label: bytes,
                  policy_encrypting_key: bytes,
                  alice_verifying_key: bytes,
-                 message_kit: bytes):
+                 message_kit: bytes,
+                 treasure_map: bytes = None):
```

Comment:
Can we use a `Union` here to specify that you can pass `bytes` or a `TreasureMap`?